src/sentry/dynamic_sampling/rules_generator.py

```diff
@@ -54,6 +55,27 @@ def generate_environment_rule() -> BaseRule:
     }
 
 
+def generate_healthcheck_rule(sample_rate: float) -> BaseRule:
+    return {
+        "sampleRate": sample_rate / HEALTH_CHECK_DROPPING_FACTOR,
+        "type": "trace",
+        "condition": {
+            "op": "or",
+            "inner": [
+                {
+                    "op": "glob",
+                    "name": "transaction.transaction",
+                    # TODO (andrii): find final list
+                    "value": ["*healthcheck*", "*healthy*"],
```

Comment:
I would suggest to generate this via a list of candidate healthcheck values so that we can easily modify them separately from the function itself.

E.g.:
```python
HEALTH_CHECKS_MATCHES = ["*healthcheck*", "*healthy*"]
...
"value": HEALTH_CHECKS_MATCHES
```

Reply:
we still not sure who final list should looks like ¯\_(ツ)_/¯

Reply:
fixed.